### ISSUE TITLE
My post does not show the complete title

### ISSUE THREAD

**Issue by nombrekeff** [SiddharthShyniben/post-view#3]
As you can see in this link:
https://siddharthshyniben.github.io/post-view/view/?link=https%3A%2F%2Fdev.to%2Fnombrekeff%2Fi-created-one-of-the-fastests-node-js-password-generators-without-knowing-16ll&title=Let

The title is only "Let", it seems it did not like the '. I just inserted the link, and the title was filled in automatically by the site.

The original title was: "Let's optimize JavaScript - password generator (2.15x faster)"
The post link is: https://dev.to/nombrekeff/i-created-one-of-the-fastests-node-js-password-generators-without-knowing-16ll

**Comment by SiddharthShyniben** [comment]
If I copy and paste the link, It works. But clicking it doesn't. I'll check it out

**Comment by SiddharthShyniben** [comment]
Yep, the problem is when I set the `href` attribute (which has a `'` character), the link breaks.

**Comment by nombrekeff** [comment]
Yup, good catch

**Comment by nombrekeff** [comment]
Great, you're a fast bug catcher 🐛
